src/commands/configure.js

```diff
@@ -6,12 +6,12 @@ const { setConfig, getConfig } = require('../config')
 class ConfigureCommand extends BaseCommand {
   async run() {
     const prompts = getPrompts(['swaggerHubUrl','apiKey'])(getConfig())
-    const { configDir } = this.config
+    const configFilePath = `${this.config.configDir}/config.json`
```

Comment:
Wrong path separator for Windows clients.
`Config saved to C:\Users\barry\AppData\Local\swaggerhub-cli/config.json`

Reply:
Updated :D